_pages/en_US/installing-boot9strap-(menuhax).txt

```diff
@@ -0,0 +1,89 @@
+---
+title: "Installing boot9strap (menuhax)"
+---
+
+{% include toc title="Table of Contents" %}
+
+{% capture technical_info %}
+<summary><em>Technical Details (optional)</em></summary>
+
+We use BannerBomb3 to install menuhax67 which lets us access the Homebrew Launcher. From there we launch uloader which launches the SafeB9SInstaller. 
+
+Info about uloader [here](https://github.com/zoogie/uloader), info about menuhax67 [here](https://github.com/zoogie/menuhax67).
+
+{% endcapture %}
+<details>{{ technical_info | markdownify }}</details>
+{: .notice--info}
+
+You should only be able to get to this page if you are running version 11.15.0 or 11.14.0. If you are on any firmware above 11.15.0, uloader will fail. 
+{: .notice--warning}
+
+### What You Need
+
+* The latest release of [SafeB9SInstaller](https://github.com/d0k3/SafeB9SInstaller/releases/download/v0.0.7/SafeB9SInstaller-20170605-122940.zip) (direct download)
+* The latest release of [boot9strap](https://github.com/SciresM/boot9strap/releases/download/1.4/boot9strap-1.4.zip) (direct download)
+* The latest release of [Luma3DS](https://github.com/LumaTeam/Luma3DS/releases/latest) (the Luma3DS `.zip` file)
+* The latest release of [BannerBomb3](https://github.com/zoogie/Bannerbomb3/releases/latest) (the RELEASE `.zip` file)
+* The latest release of [uloader](https://github.com/zoogie/uloader/releases/latest) (the `.3dsx` file)
```

Comment:
what if we direct linked these downloads... haha jk... unless?...

Reply:
???

Reply:
I mean direct linking to the actual file on github  as a direct download rather than having people click on the file in github

Reply:
ya ig wew cnas wdo it